src/css/base/_variables.scss

```diff
@@ -101,6 +101,7 @@ $coa-color-green-light: #02eaa4;
 
 $coa-color-orange: #e03400;
 $coa-color-orange-dark: #b82b00;
+$coa-color-vintage-vinilla: #FFF5C3;
```

Comment:
Vintage vinilla

Reply:
i searched and vanilla is only used here and one other place

Reply:
I hate vowels. 🙄 They're constantly torment me.

Reply:
But, it's a new color in the design. So, I thought I've give it an official name. *& fixed the spelling error.